code/modules/antagonists/fugitive/fugitive.dm

```diff
@@ -1,6 +1,8 @@
 
 /datum/antagonist/fugitive
 	name = "Fugitive"
+	antag_role_type = ROLE_KEY_FUGITIVE
+	antag_major_bancheck_type = null
```

Comment:
this should check for a ban

Reply:
`antag_major_bancheck_type = null` means they're not bannable. This means they'll not be filtered by `MAJOR_BANCHECK_ANTAGONIST` meanwhile all ghost spawns are already banchcked by `MAJOR_BANCHECK_GHOSTSPAWN`

I am not sure if fugitive things are necessary to check two major banchecks.

Reply:
Fugitives are a round antagonist now, so they should be banned like one rather than part of ghost role bans.